packages/react-hooks/src/useStashIds.ts

```diff
@@ -7,7 +7,9 @@ import useApi from './useApi';
 import useCall from './useCall';
 
 const transformStashes = {
-  transform: (stashes: AccountId[]) => stashes.map((accountId) => accountId.toString())
+  transform: (stashes: AccountId[]) => stashes
+    .filter((stashes: AccountId) => !!stashes)
+    .map((accountId) => accountId.toString())
```

Comment:
This def seems to be an issue on Edgeware, but not on Bereesheet nor other networks.  will leave this in for now, but will need to investigate what is going wrong since something is still mismatching.

Reply:
When I look at the Miannet types o GH, they still seem old, i.e. here - https://github.com/hicommonwealth/edgeware-node-types/blob/master/src/spec/mainnet.ts#L8-L24

So believe this is actually as a result of the old types used. So would suggest switching (for now) to Bereesheet types for Miannet as well, which seems to have the latest. (And then this is unneeded)

Reply:
That also doesn't quite do the trick... (In my tests, there is some mismatch somewhere)

Reply:
Ok, there is def something weird in the keys, some of them are null (this is from `api.quesry.staking.validators.keys()` as used by the derive here) -

```
[
  "ndCyA9VQAkDpw1uDedPQm9tEsJcFdnfrHNnxom3CMW9uvop",
  "ndAMaNishd8z6YcR843jdqmUUcAgCLiFhNovXDsvmgkwdba",
  "nDqE5fV5YBhCFQCt19pS3k2uDyu5ecRhE75sKxrocHnAjEp",
  "mwpdcgb8KQYwi5pCg7K174rpSZtk8QEW95VnJQaMmeaK31H",
  "hZyPPSc8iDEdmLxHoUp9cpcZavZxkVnKGWqpYeCSt5vakHe",
  "nx7Be1NK8b2uV2bSHSxDPMJqNGAr6sc2i982shToqVHBVH7",
  "kYgxuZ7Ad7vHeyN85t6PLicyKxmbE72P46iQncJJxXTKhDU",
  "hbvLTD4NR2fKziSLeH6j6sFwycHyA8ch9VRtTxiD54JkwFC",
  "mB6GJCxuDB4WzRf6yBa83eUXn7SaMTg67i54cKrKZZ6f5KY",
  null,
...
```

Reply:
Ok, investigating, doing this -

```
const keys = await api.query.staking.validators.keys();
const stashes = await api.derive.staking.stashes();
const args = keys.map((key, index) => [key.toHex(), key.args[0], stashes[index]])

console.log(JSON.stringify(args, null, 2))
```

Which yields (trimmed) -

```
...
[
    "0x5f3e4907f716ac89b6347d15ececedca88dcde934c658227ee1dfafcd6e169030f608263d3785577a20c71fe648acea8ea13e7ce0d507cfb00c1788f1eb402b756ba7dbd4dba783a",
    "mB6GJCxuDB4WzRf6yBa83eUXn7SaMTg67i54cKrKZZ6f5KY",
    "mB6GJCxuDB4WzRf6yBa83eUXn7SaMTg67i54cKrKZZ6f5KY"
  ],
  [
    "0x5f3e4907f716ac89b6347d15ececedca88dcde934c658227ee1dfafcd6e1690311320ae78b67bc3cc132f878791c063a4085531c801f4f2f2e216ebefa7ab69d",
    null,
    null
  ],
  [
    "0x5f3e4907f716ac89b6347d15ececedca88dcde934c658227ee1dfafcd6e16903171b1b5ab8e7b8f3e4b531c4f877411b4c46bc0e45e5ce94629a17b111ba715ced098b6c0116f562",
    "ngVZ5aqPr5ja2xoTYN2zVXCnPdKiXoQMbp6Ay4DAao8zJav",
    "ngVZ5aqPr5ja2xoTYN2zVXCnPdKiXoQMbp6Ay4DAao8zJav"
  ],
...
```

The ones where `null` comes through seems to be shorter than the rest, i.e. they don't seem fully encoded. Which is weird.

Reply:
Ok, see the same when I query the entries.So I'm hoping this is a once-off. Anyway, digging through to see wtf can be done.

![image](https://user-images.githubusercontent.com/1424473/94114727-08af5f00-fe49-11ea-98ce-74f5e4be0690.png)